GunShipAlpha/src/Game.java

```diff
@@ -95,8 +96,18 @@ public void spawnBehemoths(int count) {
             int side = rand.nextInt(4);
             int x = 0, y = 0;
             Game.spawnEnemiesRandomLocation result = getSpawnEnemiesRandomLocation(side, y, x, rand);
-            Enemy enemy = new Enemy(result.x(), result.y(), getWidth() / 2, getHeight() / 2);
-            gs.addEnemy(enemy);
+            Behemoth behemoth = new Behemoth(result.x(), result.y(), getWidth() / 2, getHeight() / 2);
+            gs.addBehemoth(behemoth);
+        }
+    }
+    public void spawnTanks(int count) {
+        Random rand = new Random();
+        for (int i = 0; i < count; i++) {
```

Comment:
it seems like this and spawnBehemoths both need to check to ensure that the location doesn't put the next one on top of any of the existing ones...

Reply:
fixed